packages/core/lib/modules/wallet-backup.test.js

```diff
@@ -1,16 +1,7 @@
-import {
-  WalletBackup,
-  invalidFileMessage,
-  noDocumentsFound,
-  noAccountsFound,
-} from './wallet-backup';
-import {walletService} from '../services/wallet/service';
-import {getPromiseError} from '../services/test-utils';
-import emptyBackup from '../test/fixtures/backup-files/empty-json.json';
-import emptyBackup2 from '../test/fixtures/backup-files/wallet-backup-1.json';
+import { getPromiseError } from '../services/test-utils';
+import { walletService } from '../services/wallet/service';
 import backup11 from '../test/fixtures/backup-files/wallet-backup-11.json';
```

Comment:
can we name the backup files to describe the scenario? e.g. `wallet-backup-empty-file.json`

Reply:
fixed